util/unifyfs/src/unifyfs.c

```diff
@@ -180,9 +182,11 @@ static void parse_cmd_arguments(int argc, char** argv)
 
         case 'h':
         default:
+	    printf("\n\nArgument %d is invalid!\n", argument_count);
```

Comment:
`-h` should not print this error.

Reply:
Oooh--good catch.  I'll fix that in the next commit.  Thanks.

Reply:
Fixed.